### PR TITLE
Add a solution for Finders Keepers challenge

### PR DESCRIPTION
**Pull request by nikkiselev**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers/index.md [hunk 1]
```diff
@@ -55,3 +61,27 @@ function findElement(arr, func) {
...
+```javascript
+function findElement(arr, func) {
+  return arr.find(func);
+}
+```
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/repls/FairSimultaneousBoolean' target='_blank' rel='nofollow'>Run Code</a>
+
+#### Relevant Links
+
+*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find' target='_blank' rel='nofollow'>Array.prototype.find</a>
```
Comment (RandellDawson) [inline guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers/index.md]: We prefer you to use markdown styling for the links.  See the [styling guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#links)

**Review by RandellDawson — CHANGES_REQUESTED** [review]
In addition to the change I requested in the code, please remove lines 77-79 and 85-87 as we no longer use this.  There is an outstanding PR which will remove all of these from existing challenges, but not from new challenges.

Also, we may use your solution to replace the Basic solution as it is short and simple.  We will let you know what we decide.

Thank you.

**Comment by nikkiselev** [comment]
Hi @RandellDawson , I've updated the PR. 
Thanks

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**